packages/shared/src/components/common/SpecImage.tsx

```diff
@@ -26,10 +26,6 @@ export function SpecImage({ specId, circle, size }: { specId?: string | number;
             ? `https://images.wowarenalogs.com/specs/${specId}.jpg`
             : 'data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAQAAAAnZu5uAAAAEElEQVR42mNk+M8ABYwkMAGbQQUBEvGWBAAAAABJRU5ErkJggg=='
         }
-        blurDataURL={
```

Comment:
removing this because next is throwing a warning saying these images are too small for this to be useful